doc: Adds upgrade guide for MongoDB Atlas Provider 1.30.0 (#3188)
mongodb/terraform-provider-mongodbatlas: docs/guides/1.30.0-upgrade-guide.md

---
page_title: "Upgrade Guide 1.30.0"
---

# MongoDB Atlas Provider 1.30.0: Upgrade and Information Guide

The Terraform MongoDB Atlas Provider version 1.30.0 has a number of new and exciting features.

## New Resources, Data Sources, and Features

> - You can now use `HTTPS` connections with the `mongodbatlas_stream_connection` resource. To learn more, see the [mongodbatlas_stream_connection](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_connection) resource documentation.

Reviewer: ```suggestion
- You can now use `Https` connections with the `mongodbatlas_stream_connection` resource. To learn more, see the [mongodbatlas_stream_connection](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_connection) resource documentation.
```
to follow the documentation formatting

Author: Not sure. IMO: It looks a bit ugly, but I think we prefer the consistency?

Reviewer 2: https://github.com/mongodb/terraform-provider-mongodbatlas/commit/1958e51b38c52d592abc8be0272a393ff6068b21 `Https` is how it's been defined, so yes I would prefer consistency

Author: Sounds good 👍 
Changed in a8a43c1d90e235b7dfee1775bb543ab9ba308166

> - `mongodbatlas_stream_privatelink_endpoint` now supports [Amazon Managed Streaming for Apache Kafka](https://aws.amazon.com/msk/) (AWS MSK) and exposes `error_message`, `interface_endpoint_name`, and `provider_account_id` attributes. To learn more, see the [mongodbatlas_stream_privatelink_endpoint](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_privatelink_endpoint) resource documentation.

Reviewer 3: ```suggestion
- `mongodbatlas_stream_privatelink_endpoint` now supports [Amazon Managed Streaming for Apache Kafka](https://aws.amazon.com/msk/) (AWS MSK). New `error_message`, `interface_endpoint_name`, and `provider_account_id` attributes have also been added. To learn more, see the [mongodbatlas_stream_privatelink_endpoint](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_privatelink_endpoint) resource documentation.
```

Author: Thanks. Changed in a8a43c1d90e235b7dfee1775bb543ab9ba308166

- `mongodbatlas_cloud_backup_snapshot` now supports `timeouts` attribute for the create operation.

> 1.30.0 also includes general improvements, bug fixes and several key documentation updates. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.

Reviewer 4: ```suggestion
1.30.0 also includes general improvements, bug fixes, and several key documentation updates. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more information.
```
